shuup/setting_keys.py

```diff
@@ -0,0 +1,11 @@
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2021, Shuup Commerce Inc. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+SHUUP_HOME_CURRENCY = "home_currency"
```

Comment:
move this file to `shuup/core/settings_keys.py`. You will need to split the keys into different apps: core, front, xtheme, etc.